portal/templates/initial_queries_macros.html

```diff
@@ -38,9 +51,142 @@ <h4 class="modal-title" id="termsReminderModalLabel">{{ _("To Continue") }}</h4>
+          <li>{{_("If no, thank them for their time.")}}</li>
+        </ul>
+        <div class="modal fade" id="websiteDeclarationFormModal" tabindex="-1" role="dialog">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header hidden-print">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title" id="termsReminderModalLabel">{{ _("Please print and fill out the form") }}</h4>
+              </div>
+              <div class="modal-body">
+                <div id="websiteDeclarationForm">
+                    <h4>{{_("Website declaration form")}}</h4>
+                    <br/>
+                    <p><em>{{_("Person Obtaining Consent must complete a form with the following text. This form must be entered into a central database and be retrievable in case there are any questions raised about the consent given at any time.")}}</em></p>
+                    <br/>
+                    <p><strong>{{_("Please tick the relevant box and complete the declaration below.")}}</strong><p>
+                    <p>{{_("For phone consent:")}}</p>
+                    <p><i class="fa fa-square-o fa-lg consent-form-checkbox"></i> <strong>{{_("I have read this consent form to the subject.")}}</strong></p>
+                    <br/>
+                    <p>{{_("For face-to-face consent:")}}</p>
+                    <p><i class="fa fa-square-o fa-lg consent-form-checkbox"></i> <strong>{{_("I have gone through and explained the consent form in the presence of the subject.")}}</strong></p>
+                    <br/>
+                    <p>{{_("Declaration:")}}</p>
+                    <p><strong>{{_("An explanation of the TrueNTH website tool was given and questions from the subject were solicited and answered to the subject’s satisfaction. In my judgment, the subject demonstrated comprehension of the information. The subject has provided oral consent to participate in the IRONMAN Registry Study on the terms set out above.")}}</strong></p>
+                    <br/><br/>
+                    <div class="box-consent-container">
+                    {{_("Name and Title")}}
+                    </div>
+                    <br/><br/>
+                    <div class="box-consent-container">
+                    {{_("Date")}}
+                    </div>
+                    <br/><br/>
+                    <div class="box-consent-container">
+                    {{_("Signature of Person Obtaining Consent")}}
+                    </div>
+                    <br/><br/>
+                </div>
+              </div>
+              <div class="modal-footer hidden-print">
+                <button type="button"  class="btn btn-default" id="consentPrintButton" onclick="window.print()">{{_("PRINT") }}</button>
```

Comment:
I think translations are case-sensitive, might wanna use the `upper` filter to avoid duplicating strings if possible eg
```
{{_("Print")|upper }}
```